### ISSUE TITLE
add TERT promoter mutations to TCGA studies

### ISSUE THREAD

**Issue by tmazor** [cBioPortal/datahub#154]
Some TCGA studies have called TERT promoter mutation but we don't have that data in cBioPortal. This is likely because the mutations may not come from the standard exome sequencing pipeline but from an independent sequencing assay specifically targeted at the TERT promoter.

We need to identify which studies have TERT mutation data and add that data to cBioPortal.

Below is an incomplete list of studies with TERT promoter mutation data:
- [ ] LGG/GBM https://www.ncbi.nlm.nih.gov/pubmed/26824661
- [ ] LGG https://www.ncbi.nlm.nih.gov/pubmed/26061751
- [ ] Cutaneous Melanoma https://www.ncbi.nlm.nih.gov/pubmed/26091043
- [ ] Hepatocellular Carcinoma https://www.ncbi.nlm.nih.gov/pubmed/28622513

**Comment by tmazor** [comment]
A user has requested the Cutaneous Melanoma data specifically: https://groups.google.com/forum/#!topic/cbioportal/jdRjXLsnlzk

**Comment by tmazor** [comment]
@jjgao @ritikakundra @yichaoS any update on when this data will be made available? I just received another user request about this data, specifically for LGG and GBM.

**Comment by kojix2** [comment]
Hello.

I use cBioPortal every day. The service is really great.

https://www.ncbi.nlm.nih.gov/pubmed/28622513
> TERT promoter mutations were the most common somatic mutation, found in 87 of 196 (44%) HCCs analyzed in the TERT promoter region (Figure S1A, Table S3). 

I would like to access the most frequent mutations in liver cancer through the cBioPortal.

Thanks!

**Comment by laveniyas** [comment]
CBioPortal is an excellent resource, thank you. Just wondering, has TERT promoter status been added to cBioPortal, especially for GBM?

**Comment by tmazor** [comment]
we should still do this

**Comment by ritikakundra** [comment]
@sbabyanusha @rmadupuri need to prioritize this cared

**Comment by rmadupuri** [comment]
Looked into the publications for TERT variants. Few studies sequenced for TERTp but the data is not available. Will reach out to the authors.

<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:////Users/madupurr/Library/Group%20Containers/UBF8T346G9.Office/TemporaryItems/msohtmlclip/clip.htm">
<link rel=File-List
href="file:////Users/madupurr/Library/Group%20Containers/UBF8T346G9.Office/TemporaryItems/msohtmlclip/clip_filelist.xml">



</head>

<body link="#0563C1" vlink="#954F72">



Study | TERTp Sequencing Status
-- | --
blca_tcga_pub | No
blca_tcga_pub_2017 | No
brca_tcga_pub | No
brca_tcga_pub2015 | No
coadread_tcga_pub | No
gbm_tcga_pub | No
gbm_tcga_pub2013 | Yes https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3910500/. (Look at Fig   4C. 25 samples analysed for TERTp. 21 C228T, 6 C250T, 4 WT - but no data   available)
hnsc_tcga_pub | No
kich_tcga_pub | No
kirc_tcga_pub | No
laml_tcga_pub | No
lgggbm_tcga_pub | Yes but data not available
lusc_tcga_pub | No
luad_tcga_pub | No
ov_tcga_pub | No
pcpg_tcga_pub | No
prad_tcga_pub | 19 samples were high pass whole genome sequenced for TERTp. No TERTp   variants found. No info on which 19 samples were sequenced.
sarc_tcga_pub | TERTp validation was done. But no data available. Portal has 1 TERTp variant for the study. Looks like the data was already included in the MAF.
skcm_tcga_pub_2015 | Yes. Data added to portal
stad_tcga_pub | No
stes_tcga_pub | No
thca_tcga_pub | Yes. Data added to portal
ucec_tcga_pub | No
Hepatocellular Carcinoma | Yes. Has TERTp data. Study not curated to portal yet. https://pubmed.ncbi.nlm.nih.gov/28622513/



</body>

</html>

**Comment by santiago-es** [comment]
Hi, thanks for adding this data to CBIO portal. I noticed some of the studies above had classified TERTp but had not posted the data in portal. I had spent some time tthree years ago annotating the patient IDs per study by hand for papers that had published this info, attached. Just in case it's useful, may be redundant. 

[TCGA TERTp Patient IDs.xlsx](https://github.com/cBioPortal/datahub/files/14780125/TCGA.TERTp.Patient.IDs.xlsx)
